app/helpers/hosts_and_hostgroups_helper.rb

```diff
@@ -103,6 +103,7 @@ def realm_field(f, can_override = false, override = false)
   def interesting_klasses(obj)
     classes    = obj.all_puppetclasses
     classes_ids = classes.reorder('').pluck('puppetclasses.id')
+    p "interesting_klasses"
```

Comment:
Hehe, forgot this debugging p :)
On Sep 2, 2015 1:20 PM, "Daniel Lobato García" <notifications@github.com>
wrote:

> In app/helpers/hosts_and_hostgroups_helper.rb
> <https://github.com/theforeman/foreman/pull/2664#discussion_r38517157>:
>
> > @@ -103,6 +103,7 @@ def realm_field(f, can_override = false, override = false)
> >    def interesting_klasses(obj)
> >      classes    = obj.all_puppetclasses
> >      classes_ids = classes.reorder('').pluck('puppetclasses.id')
> > +    p "interesting_klasses"
>
> [image: :question:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/theforeman/foreman/pull/2664/files#r38517157>.
>